Error on "go get github.com/qntfy/kazaam"

Hi,

in the master branch of `github.com/satori/go.uuid`, `uuid.NewV4()` returns an error. In the pinned version `v1.2.0`, that function does not return an error. But since `go get` always pulls the master branch, `github.com/qntfy/kazaam` is not `go get`-able at the moment.

Here's the error:
`../github.com/qntfy/kazaam/transform/uuid.go:33:18: multiple-value uuid.NewV4() in single-value contex`

This problem was introduced with PR #76 .

--- a/glide.yaml
+++ b/glide.yaml
@@ -2,5 +2,5 @@ package: github.com/qntfy/kazaam
 import:
 - package: github.com/qntfy/jsonparser
   version: ^1.0.2
-- package: github.com/satori/go.uuid
-  version: ^1.2.0
+- package: github.com/gofrs/uuid
+  version: ^3.1.0

--- a/transform/uuid.go
+++ b/transform/uuid.go
@@ -3,7 +3,7 @@ package transform
 import (
 	"strings"
 
-	uuid "github.com/satori/go.uuid"
+	uuid "github.com/gofrs/uuid"
 )
 
 var (
@@ -30,7 +30,10 @@ func UUID(spec *Config, data []byte) ([]byte, error) {
 
 		switch version {
 		case 4:
-			u = uuid.NewV4()
+			u, err = uuid.NewV4()
+			if err != nil {
+				return nil, err
+			}
 
 		case 3, 5:
 			// choose the correct UUID function
@@ -57,7 +60,7 @@ func UUID(spec *Config, data []byte) ([]byte, error) {
 			// generate the required namespace
 			u, err = namespaceFromString(namespaceString)
 			if err != nil {
-				return nil, SpecError("namespace is not a valid UUID or is not DNS, URL, OID, X500")
+				return nil, SpecError("Namespace is not a valid UUID or is not DNS, URL, OID, X500")
 			}
 
 			// loop over the names field
@@ -80,7 +83,7 @@ func UUID(spec *Config, data []byte) ([]byte, error) {
 			return nil, versionError
 
 		}
-		// set the uuid in the appropraite place
+		// set the uuid in the appropriate place
 		data, err = setJSONRaw(data, bookend([]byte(u.String()), '"', '"'), k)
 		if err != nil {
 			return nil, err

--- a/transform/uuid_test.go
+++ b/transform/uuid_test.go
@@ -3,8 +3,8 @@ package transform
 import (
 	"testing"
 
+	uuid "github.com/gofrs/uuid"
 	"github.com/qntfy/jsonparser"
-	uuid "github.com/satori/go.uuid"
 )
 
 func TestUUIDV4(t *testing.T) {

